modules/claims_api/config/schemas/v2/526.json

```diff
@@ -543,7 +543,7 @@
           "name": {
             "description": "Name of the disability. The /disabilities endpoint on the [Benefits Reference Data API](https://developer.va.gov/explore/benefits/docs/benefits_reference_data?version=current) may be used to retrieve possible disability names.",
             "type": "string",
-            "pattern": "^$|([a-zA-Z0-9\"\\/&\\(\\)\\-'.,# ]([a-zA-Z0-9(\\)\\-'.,# ])?)+$",
+            "pattern": "^$|([a-zA-Z0-9\"\\/&\\(\\)\\-'.,#\\[\\] ]([a-zA-Z0-9(\\)\\-'.,#\\[\\] ])?)+$",
```

Comment:
@tycol7 It appears the Docker Container does not accept the name with brackets so I believe the `disability_compensation_evss_mapper` will also need to be updated to remove the brackets from the name.
  
<img width="894" alt="Screenshot 2024-08-15 at 9 28 28 AM" src="https://github.com/user-attachments/assets/7b74fb0c-c3f1-4ac8-a451-f5613ee4344c">

When I removed the brackets via the mapper using `disability[:name] = disability[:name].gsub(/\[|\]/, '')
` the claim established as expected.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/18037/commits/4c4af5887e87492cd5d4764540b53ff94761827b